frontend/components/ui/navbar.tsx

```
@@ -0,0 +1,225 @@
+                                </button>
+                            </SheetTrigger>
+
+                            <SheetContent className='w-3/5 bg-white'>
+                                <SheetHeader>
+                                    <Image
+                                        src='https://skillgro-react.netlify.app/assets/img/logo/logo.svg'
+                                        alt='Logo'
+                                        width={153}
+                                        height={38}
+                                    />
+                                    <SheetTitle className='hidden'></SheetTitle>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)